src/core/Form/Checkbox/CheckboxGroup.baseStyles.tsx

```diff
@@ -0,0 +1,16 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+export const baseStyles = (theme: SuomifiTheme) => css`
+  ${element(theme)}
+  ${font(theme)('bodyText')}
+
+  & .fi-checkbox_container {
+    margin-top: ${theme.spacing.xs};
+  }
+
```

Comment:
In the designs there's a 10px margin between the label and help text. Should it be added here as well or was something else decided on it?

Reply:
Not intentionally denied to left out. Was creating this based on the RadiobuttonGroup and went the same road for not telling the explicit margin between label and hint text.

I will add the margin.

Reply:
Margin added.

The logic for showing the margin between label, hint text and between actual group was added.
Also the large variant in Checkboxes have different margin, so therefore it uses also top margin to take care of that.